guide/lib/examples/footer_helpers.rb

```diff
@@ -34,10 +34,8 @@ def footer_meta_complex_items
     def footer_with_custom_meta_html
       <<~FOOTER_META_HTML
         = render GovukComponent::FooterComponent.new do |footer|
-          - footer.meta_html do
-            .govuk-footer__meta-custom class="govuk-\!-margin-top-1"
-              | Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam
-                a porta purus. Fusce faucibus aliquam massa sed eleifend.
+          - footer.with_meta_html do
+            | Built with love by x-govuk.
```

Comment:
Might be useful to show including a link in this example (which would need the `govuk-footer__link` applied); not sure what the correct way of doing that would be, off the top of my heading, but I guess using the `govuk_link` helper.

```suggestion
            | Built with love by X-GOVUK.
```

Reply:
Good point. I think there's room for a `#govuk_footer_link_to` helper as the ones in the example don't have the `govuk-link` class.